docs/src/man/sorting.md

```diff
@@ -5,7 +5,8 @@ Sorting is a fundamental component of data analysis. Basic sorting is trivial: j
 ```jldoctest sort
 julia> using DataFrames, CSV
 
-julia> iris = CSV.read(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv"));
+julia> iris = CSV.File(joinpath(dirname(pathof(DataFrames)), "../docs/src/assets/iris.csv")) |>
```

Comment:
Better use the more common syntax `DataFrame(...)`? That will avoid confusing newcomers.

Reply:
fixed (although in my codes I use the `File` pattern 😄)

Reply:
No, I meant using `DataFrame(CSV.File(...))` instead of `|>`.

Reply:
OK. Let us stick with `CSV.read` with `copycols=true` (what I pushed now) as it is also a pattern that is good to show I think.

Reply:
I prefer `DataFrame(CSV.File(...))`, as it makes it less likely that people will call `CSV.read` and forget to pass `copycols=true`. Actually I'd rather get rid of `CSV.read`.

BTW, there are other occurrences in the docs IIRC, better change all of them at the same time.

Reply:
OK - done. At least we agree what should be used 😄.